unittests/fixtures/factory/web-area/monitor/entry_el7_osg35

```diff
@@ -0,0 +1 @@
+entry_el7_osg34
```

Comment:
Should this be `..._osg35` instead because I see the file is named `entry_el7_osg35`?

Reply:
I think it is fine, the link points to the existing file, it does not seem broken. .. _osg35 would be self-referencing
```
drwxr-xr-x   3 marcom  admin   96 Feb  4  2022 entry_el7_osg34
lrwxr-xr-x@  1 marcom  admin   15 Mar 22 17:38 entry_el7_osg35 -> entry_el7_osg34
lrwxr-xr-x@  1 marcom  admin   15 Mar 22 17:39 entry_el8_osg34 -> entry_el8_osg35
drwxr-xr-x   3 marcom  admin   96 Feb  1  2024 entry_el8_osg35
```